web-app/src/app/screens/Feed/index.tsx

```diff
@@ -132,7 +132,7 @@ export default function Feed(): React.ReactElement {
   const datasets = useSelector(selectDatasetsData);
   const hasLoadedAllDatasets = useSelector(selectHasLoadedAllDatasets);
   const latestDataset = useSelector(selectLatestDatasetsData);
-  const latestGbfsVersion = useSelector(selectLatestGbfsVersion);
+  const latestGbfsVersion = useSelector(selectAutodiscoveryGbfsVersion);
```

Comment:
[question] In terms of naming, would the GbfsVersion of source=autodiscovery be the latest? In the array of versions, could there only be 1 of source = 'autodiscovery'?